### PR TITLE
Create Cilium org README.md

### PR DESCRIPTION
**Pull request by xmulligan**
Signed-off-by: Bill Mulligan <billmulligan516@gmail.com>

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+## Hey, this is Cilium 🐝🐝🐝
+
+[![Our](https://img.shields.io/static/v1?label=Our&message=Website&color=blue)](https://cilium.io/)
+[![Sign-up](https://img.shields.io/static/v1?label=Sign-up&message=for%20news&color=red)](https://cilium.io/newsletter)
+[![Contributing](https://img.shields.io/static/v1?label=Contributing&message=Guide&color=brightgreen)](https://cilium.io/newsletter)
```
Comment (lizrice) [inline profile/README.md]: Maybe link to Get Involved? 

 ```suggestion
[![Contributing](https://img.shields.io/static/v1?label=Contributing&message=Guide&color=brightgreen)](https://cilium.io/get-involved)
```

File: profile/README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+[![Roadmap](https://img.shields.io/static/v1?label=Roadmap&message=public&color=blueviolet)](https://docs.cilium.io/en/latest/community/roadmap/)
+[![Governance](https://img.shields.io/static/v1?label=Governance&message=and%20Maintainers&color=yellow)](https://github.com/cilium/cilium/blob/master/MAINTAINERS.md)
```
Comment (lizrice) [inline profile/README.md]:
```suggestion
[![Governance](https://img.shields.io/static/v1?label=Governance&message=and%20Maintainers&color=yellow)](https://docs.cilium.io/en/stable/contributing/governance/)
```

Suggest the docs page as it includes a link to the maintainers but also the broader background

File: profile/README.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+<img src="https://camo.githubusercontent.com/714c5d777b0025dda66b46f14e28badc01e3e3360ef264be204f54846a7c9573/68747470733a2f2f63646e2e6a7364656c6976722e6e65742f67682f63696c69756d2f63696c69756d406d61737465722f446f63756d656e746174696f6e2f696d616765732f63696c69756d5f6f766572766965772e706e67" alt="Cilium overview diagram" width="800">
```
Comment (lizrice) [inline profile/README.md]: For consistency with the other sections I wonder if this picture should go into the Cilium section (so there's a diagram for Cilium, Hubble & Tetragon)

File: profile/README.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+### Cilium
+
+Cilium is used to provide and transparently secure network connectivity and load balancing between application workloads such as application containers, processes, or VMs. Cilium operates at Layer 3/4 to provide traditional networking and security services as well as Layer 7 to protect and secure use of modern application protocols such as HTTP, gRPC, and Kafka. Cilium is a part of the [Cloud Native Computing Foundation](https://www.cncf.io/) and is their most advanced and widely used CNI for Kubernetes. Check out the [Getting Started Guides](https://docs.cilium.io/en/stable/gettingstarted/) to try it out.
+
+Cilium has hundreds of [adopters](https://cilium.io/adopters) around the world including Bell Canada, Capital One, Google, The New York Times, and Yahoo. If you are using Cilium, [add yourself](https://github.com/cilium/cilium/blob/master/USERS.md) to the list today!
+
+Over 400 people have already contributed to the Cilium project and [you can too](https://docs.cilium.io/en/stable/contributing/development/contributing_guide/).
```
Comment (lizrice) [inline profile/README.md]: Wonder if this sentence about contributing should go under Community section? To make it clearer that people can contribute in all three projects. I don't feel strongly about it though

**Review by lizrice — COMMENTED** [review]
A few little suggestions but generally looks good!

File: profile/README.md (hunk 3 above)
Comment (xmulligan) [inline profile/README.md]:
```suggestion
```

File: profile/README.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+### Cilium
+
+Cilium is used to provide and transparently secure network connectivity and load balancing between application workloads such as application containers, processes, or VMs. Cilium operates at Layer 3/4 to provide traditional networking and security services as well as Layer 7 to protect and secure use of modern application protocols such as HTTP, gRPC, and Kafka. Cilium is a part of the [Cloud Native Computing Foundation](https://www.cncf.io/) and is their most advanced and widely used CNI for Kubernetes. Check out the [Getting Started Guides](https://docs.cilium.io/en/stable/gettingstarted/) to try it out.
+
+Cilium has hundreds of [adopters](https://cilium.io/adopters) around the world including Bell Canada, Capital One, Google, The New York Times, and Yahoo. If you are using Cilium, [add yourself](https://github.com/cilium/cilium/blob/master/USERS.md) to the list today!
+
```
Comment (xmulligan) [inline profile/README.md:18]:
```suggestion

<img src="https://camo.githubusercontent.com/714c5d777b0025dda66b46f14e28badc01e3e3360ef264be204f54846a7c9573/68747470733a2f2f63646e2e6a7364656c6976722e6e65742f67682f63696c69756d2f63696c69756d406d61737465722f446f63756d656e746174696f6e2f696d616765732f63696c69756d5f6f766572766965772e706e67" alt="Cilium overview diagram" width="800">
```

File: profile/README.md (hunk 4 above)
Comment (xmulligan) [reply profile/README.md]: good idea, I'll move it

Comment (xmulligan) [inline profile/README.md]:
```suggestion
```

File: profile/README.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+* System call activity
+* I/O activity including network & file access
+
+When used in a Kubernetes environment, Tetragon is Kubernetes-aware - that is, it understands Kubernetes identities such as namespaces and pods - so that security event detection can be configured for specific individual workloads.
+
+<img src="https://github.com/cilium/tetragon/raw/main/docs/images/smart_observability.png" alt="Tetragon Overview Diagram" width="800">
+
+### Community
+
+Cilium is an open source project that anyone in the community can use, improve, and enjoy. We'd love you to join us! Here's a few ways to find out what's happening and get involved:
```
Comment (xmulligan) [inline profile/README.md]:
```suggestion
Cilium is an open source project that anyone in the community can use, improve, and enjoy. Over 400 people have already contributed to the Cilium project and [you can too](https://docs.cilium.io/en/stable/contributing/development/contributing_guide/). We'd love you to join us! Here's a few ways to find out what's happening and get involved:
```

File: profile/README.md (hunk 3 above)
Comment (xmulligan) [reply profile/README.md]: moved

**Review by aanm — APPROVED** [review]

**Pull request merged**